src/containers/profile/index.jsx

```diff
@@ -32,17 +33,17 @@ export const Profile = () => {
 
   return (
     <section className={profile}>
-      <section className={mobile_burger}>
+      <div className={mobile_burger}>
         <button onClick={() => setMenu({ marginLeft: '0' })}>
           <div className={photo} />
         </button>
-      </section>
-      <section className={mobile_menu} style={menus}>
-        <section className={user}>
+      </div>
+      <div className={mobile_menu} style={menus}>
+        <div className={user}>
```

Comment:
rewrite to the `<ul> <li>` list of items.

Reply:
I'm use simple structure for 4 elements like this:
```
 <div>
  <button .../>
  <button .../>
  <button .../>
  <button .../>
</div>
```
This is excess code if I'll change that block like:
```
<div>
   <ul>
      <li>
        <button .../>
      </li>
      <li>
        <button .../>
      </li>
      <li>
        <button .../>
      </li>
      <li>
        <button .../>
      </li>
   </ul>
</div>
```